.github/workflows/main.yml

```diff
@@ -61,6 +61,9 @@ jobs:
       shell: bash
 
     - run: cargo test --all
+    - run: cargo test --all
+      env:
+        RUSTFLAGS: --cfg=check_exhaustive
```

Comment:
Total nitpicking, but: since this isn't "namespaced" as a cargo feature, perhaps it should be `--cfg=wast_check_exhaustive`?